src/prompt/cursor.rs

```diff
@@ -34,6 +55,24 @@ impl StringCursor {
         }
     }
 
+    pub fn move_word_left(&mut self) {
```

Comment:
A bit misleading name, I thought it actually moves the words. Maybe this `move_left_by_word` should sound less ambiguous.

Reply:
done in https://github.com/fadeevab/cliclack/pull/9/commits/ae9b394a0abdf057888b3e3ab0a84dd488f20318